src/stack_pr/cli.py

```diff
@@ -911,25 +963,32 @@ def command_submit(
 
     # Create local branches and initialize base and head fields in the stack
     # elements
-    init_local_branches(st, args.remote, args.verbose, args.branch_name_template)
+    init_local_branches(
+        st,
+        args.remote,
+        verbose=args.verbose,
+        branch_name_template=args.branch_name_template,
+    )
     set_base_branches(st, args.target)
-    print_stack(st, args.hyperlinks)
+    print_stack(st, links=args.hyperlinks, level=1)
```

Comment:
IMO it's better to not specify the level explicitly. It isn't necessary, doesn't improve readability, and doesn't really prevent any errors in this case.

Reply:
done